package one.clproto.logging;

enum LogLevel {
  LDEBUG = 0;
  INFO = 1;
  WARNING = 2;
  ERROR = 3;
  FATAL = 4;
  NONE = 5;
}

message ChangeRemoteLogLevel {
  required LogLevel level = 1;
}

message LogMessage {
  required LogLevel level = 1;
  required uint32 pid = 2;
  required string file_name = 3;
  required uint32 line = 4;
  required uint64 timestamp = 5;
  required string message = 6;
}
